Yona/Yona/Welcome/WelcomeViewController.swift

```diff
@@ -118,4 +118,9 @@ class WelcomeViewController: BaseViewController {
         environmentChangedAlert.addAction(cancelAction)
         self.present(environmentChangedAlert, animated: true, completion: nil)
     }
+    
+    func removeWhitespaceFromURL(url: String) -> String {
+        let newURL = url.replacingOccurrences(of: " ", with: "")
```

Comment:
Join this line with the next one, like this:
```
return url.replacingOccurrences(of: " ", with: "")
```

Reply:
done